### PR TITLE
Add an acknowledgements section.

### PR DESCRIPTION
**Pull request by hober**
The initial commit is of an acks section based solely on the GitHub commit history of the document. I'm going to go through all the closed and open issues and PRs too, so this isn't yet ready to land.

***
<a href="https://pr-preview.s3.amazonaws.com/w3ctag/security-questionnaire/pull/134.html" title="Last updated on Dec 9, 2021, 5:32 PM UTC (5f6bdae)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/security-questionnaire/134/e1e9c93...5f6bdae.html" title="Last updated on Dec 9, 2021, 5:32 PM UTC (5f6bdae)">Diff</a>

***
<a href="https://pr-preview.s3.amazonaws.com/w3ctag/security-questionnaire/pull/134.html" title="Last updated on Mar 23, 2022, 8:26 PM UTC (b80f7f2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/security-questionnaire/134/e1e9c93...b80f7f2.html" title="Last updated on Mar 23, 2022, 8:26 PM UTC (b80f7f2)">Diff</a>

### CODE REVIEW

**Comment by hober** [comment]
Okay, I've finished going through the issues and PRs. Please take a look, @torgo & @pes10k!

File: index.bs [hunk 1]
```diff
@@ -11,6 +11,7 @@ Editor: Pete Snyder, w3cid 109401, Brave https://brave.com, psnyder@brave.com
 Former Editor: Jason Novak, Apple Inc., https://apple.com
 Former Editor: Lukasz Olejnik, Independent researcher, https://lukaszolejnik.com
 Former Editor: Mike West, Google Inc., mkwst@google.com
+Former Editor: Yan Zhu, Yahoo Inc., yzhu@yahoo-inc.com
```
Comment (pes10k) [inline index.bs:14]: I can ask Yan what email she prefers, but yan@brave.com might be better (or at least more current)

**Review by pes10k — APPROVED** [review]
I left a comment around Yan's email address, but besides that, looks good to me!

**Comment by pes10k** [comment]
extremely minor issue too, but while you're editing the editors section, could you change my attribution to 

`Peter Snyder, w3cid 109401, Brave Software https://brave.com, pes@brave.com`?

Happy to do so in a separate PR too if ya prefer

File: index.bs (hunk 1 above)
Comment (diracdeltas) [reply index.bs:14]: i'm yan@brave.com, thanks!

Comment (hober) [reply index.bs:14]: Yes, you're at Brave now, but when you were an editor you were at Yahoo, yeah? I think this should reflect the state at the time. Like, if @mikewest ever leaves Google, we wouldn't retroactively change his affiliation here.

**Comment by hober** [comment]
> extremely minor issue too, but while you're editing the editors section, could you change my attribution

Done, in 5f6bdae.

File: index.bs (hunk 1 above)
Comment (hober) [reply index.bs]: Fixed in b80f7f2.

**Comment by hober** [comment]
verbal r+ from @pes10k at San Diego F2F; merging.

**Pull request merged**